src/plugins/EmojiTrigger.tsx

```diff
@@ -0,0 +1,92 @@
+import { InputRule } from "prosemirror-inputrules";
```

Comment:
BlockMenuTrigger and EmojiTrigger has lot of copy/paste code, but i not sure to create a common plugin or common trigger extension